mapviz_plugins/include/mapviz_plugins/point_draw.h

```diff
@@ -0,0 +1,115 @@
+// *****************************************************************************
+//
+// Copyright (c) 2014, Southwest Research Institute® (SwRI®)
+// All rights reserved.
+//
+// Redistribution and use in source and binary forms, with or without
+// modification, are permitted provided that the following conditions are met:
+//     * Redistributions of source code must retain the above copyright
+//       notice, this list of conditions and the following disclaimer.
+//     * Redistributions in binary form must reproduce the above copyright
+//       notice, this list of conditions and the following disclaimer in the
+//       documentation and/or other materials provided with the distribution.
+//     * Neither the name of Southwest Research Institute® (SwRI®) nor the
+//       names of its contributors may be used to endorse or promote products
+//       derived from this software without specific prior written permission.
+//
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+// AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+// IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
+// ARE DISCLAIMED. IN NO EVENT SHALL <COPYRIGHT HOLDER> BE LIABLE FOR ANY
+// DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+// (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+// LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND
+// ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+// (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+// SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// *****************************************************************************
+
+#ifndef MAPVIZ_POINT_DRAW_H_
+#define MAPVIZ_POINT_DRAW_H_
+
+// C++ standard libraries
+#include <string>
+#include <list>
+
+#include <mapviz/mapviz_plugin.h>
+
+// QT libraries
+#include <QGLWidget>
+#include <QObject>
+#include <QWidget>
+
+// ROS libraries
+#include <ros/ros.h>
+#include <tf/transform_datatypes.h>
+
+#include <mapviz/map_canvas.h>
+namespace mapviz_plugins
+{
+  class PointDraw : public mapviz::MapvizPlugin
```

Comment:
I'm not sure I like the name "PointDraw" for a class, probably because it's not a noun.  Why not something like "PointDrawingPlugin"?